src/run.js

```diff
@@ -283,7 +284,9 @@ const processPage = ({
 
       let response;
 
-      if (withoutjavascript) {
+	  if (!isJavaScriptEnabled) {
+		page.setJavaScriptEnabled(false);
```

Comment:
Not only is this missing the `await ` but I overall don't understand this change. 

By default, `minimalcss` will run the page 1 time; with JavaScript enabled. 
If you pass `options.withoutjavascript = true` it will run the page 2 times; one time without javascript and one time with. 

So what does `isJavascriptEnabled` do?

Reply:
If you pass isJavascriptEnabled = false it will run the page 1 time without javascript, because in some cases styles inserts dynamically by js, but i don't need them

Reply:
But `minimalcss` already has that functionality.

Reply:
How can i get this?

Reply:
Let's start over. What is your whole purpose of this? Do you want the minimalcss ONLY when javascript is completely disabled?

Reply:
Yes